spec/models/post_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is valid with a title' do
+    user = User.create(name: 'Roy Batty', posts_counter: 5)
+    post = Post.new(title: 'Valid Title', author: user)
+    expect(post).to be_valid
+  end
+
+  it 'is not valid without a title' do
+    user = User.create(name: 'Roy Batty', posts_counter: 5)
+    post = Post.new(author: user)
+    expect(post).not_to be_valid
+  end
+
+  it 'is not valid with a title > 250 characters' do
+    user = User.create(name: 'Roy Batty', posts_counter: 5)
+    post = Post.new(title: 'A' * 251, author: user)
+    expect(post).not_to be_valid
+  end
+
+  it "increments author's posts_counter after create" do
+    user = User.create(name: 'Roy Batty', posts_counter: 5)
+    Post.create(title: 'Sample Title', author: user)
+    expect(user.reload.posts_counter).to eq(6)
+  end
```

Comment:
- Good job done so far, but according to the project requirement, you are to test all customer methods in your model. Please write tests for the `increment_post_counter` and `decrement_post_counter`. Please fix this 🙏 
- Also, please write tests for the increment and decrement posts_comment_counter in the comment model, please fix this to ensure the methods in your models are tested according to the project requirements 😄 
- Also, please write tests for the increment and decrement likes_counter in the likes model, please fix this to ensure the methods in your models are tested according to the project requirements 😄